From c717122cecf949e3c9759ecc3ff7f5529ba5dadf Mon Sep 17 00:00:00 2001
From: Dingxian Wen <shawn.wen@rock-chips.com>
Date: Fri, 7 Aug 2020 15:14:36 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable CONFIG_VIDEO_TC35874X

Enable tc35874x hdmi to mipi CSI-2 bridge driver for hdmi-in application.

Signed-off-by: Dingxian Wen <shawn.wen@rock-chips.com>
Change-Id: Ibd9c2bd50fdde06c6c3f238b3627e071bcf50c08
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 326a9a3ad416..1f4e228436d6 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -552,6 +552,7 @@ CONFIG_VIDEO_GC032A=y
 CONFIG_VIDEO_GC2145=y
 CONFIG_VIDEO_GC2355=y
 CONFIG_VIDEO_SGM3784=y
+CONFIG_VIDEO_TC35874X=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
 CONFIG_DRM_IGNORE_IOTCL_PERMIT=y
-- 
2.35.3

